components/audit_table/audit_row/audit_row.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Would be great to have some unit tests for this component. We are passing to adding a new component, so we should be testing it, specially because the old tests are no longer verifying the right content of this new component. At least some snapshot testing is needed here.